core/src/main/scala/Elo.scala

```diff
@@ -36,6 +35,14 @@ object Elo extends OpaqueInt[Elo]:
   def computeRatingDiff(player: Player, games: Seq[Game]): Int =
     computeNewRating(player, games) - player.rating
 
+  def getExpectedScore(ratingDiff: Int): Double =
+    val absRatingDiff = ratingDiff.abs
+    val expectedScore = conversionTableFIDE.collectFirst {
+      case (range, pd) if range.contains(absRatingDiff) => pd
+    }.get
```

Comment:
We shouldn't use get here, We can use `fold` or `getOrEsle(defaultExpectedScore)` instead